### PR TITLE
Expose BuilderProcess to the API

### PR DESCRIPTION
**Pull request by EmotionalLove**
Exactly what the title says UwU

### CODE REVIEW

File: src/api/java/baritone/api/process/IBuilderProcess.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ */
+public interface IBuilderProcess extends IBaritoneProcess {
+
+    /**
+     * Build from a schematic file inside of .minecraft/schematics
+     *
+     * @param schematicFile The name of the schematic file
+     * @return Success or failure
+     */
+    boolean build(String schematicFile);
```
Comment (ZeroMemes) [inline src/api/java/baritone/api/process/IBuilderProcess.java]: This provides very limited usage of the builder function, you can't use a custom schematic type, custom naming, or custom origin.

File: src/main/java/baritone/Baritone.java [hunk 2]
```diff
@@ -1,219 +1,220 @@
-/*
```
Comment (ZeroMemes) [inline src/main/java/baritone/Baritone.java]: I don't even know what to say about this, what could you possibly do to remove and re-add the whole file.

File: src/api/java/baritone/api/IBaritone.java [hunk 3]
```diff
@@ -35,6 +32,13 @@
  */
 public interface IBaritone {
 
+    /**
+     * @return The {@link IBuilderProcess} instance
+     * @see IBuilderProcess
+     */
+    IBuilderProcess getBuilderProcess();
```
Comment (ZeroMemes) [inline src/api/java/baritone/api/IBaritone.java]: This method should align in order with the Baritone implementation

File: src/main/java/baritone/process/BuilderProcess.java [hunk 4]
```diff
@@ -82,6 +86,13 @@ public boolean build(String schematicFile) {
         return true;
     }
 
+
+    @Override
+    public void clearArea(int clearX, int clearY, int clearZ) {
+        this.build("cleararea", new AirSchematic(clearX, clearY, clearZ), ctx.playerFeet());
```
Comment (ZeroMemes) [inline src/main/java/baritone/process/BuilderProcess.java]: Not a friendly name for the user

**Comment by ZeroMemes** [comment]
On top of the review comments I made, your commit is not linked to your GitHub account, it isn't signed, and I've already decided to make the changes myself.

**Comment by wine** [comment]
Don't ever say UwU in this christian repo again, thanks.

**Comment by leijurv** [comment]
uwu motherfucker

**Comment by wine** [comment]
No!

**Comment by leijurv** [comment]
uwu uwu uwu uwu!!

**Comment by wine** [comment]
Quit it!

**Comment by leijurv** [comment]
*growls while slowly exposing my terrifying eagle beak*

**Comment by wine** [comment]
Stop erping in the Baritone repo, mr leijurv

**Comment by leijurv** [comment]
*rips you to shreds with my sharp eagle talons killing you instantly*

*this repo is my turf, or rather, nest*

*screeches warning you to stay away*

**Comment by wine** [comment]
I am requesting ownership over this repo as I consider you being too mentally unstable to be the main leader of the project.

**Comment by leijurv** [comment]
you have an anime profile pic so your opinion doesn't count *chuckles through beak*

**Comment by wine** [comment]
Well played

**Comment by leijurv** [comment]
*literally triumphantly eating your corpse*

**Comment by EmotionalLove** [comment]
Uh

**Comment by EmotionalLove** [comment]
Wow Brady hates me kms

**Pull request closed without merging**